draft-ietf-core-sid.md

```diff
@@ -738,6 +738,37 @@ The following Activity diagram summarizes the update of a YANG module and its as
 ~~~~
 {: #fig-sid-file-update title="YANG and \".sid\" file update" align="left"}
 
+# Keeping a SID File in a YANG Instance Data file
+
+{{?RFC9195}} defines a format for "YANG Instance Data".
+This essentially leads to an encapsulation of the instance data within
+some metadata envelope.
+
+If a SID file needs to be stored in a YANG Instance Data file, this
+can be achieved by embedding the value of the SID file as the value of the
+`content-data` member in the following template, and copying over the
+second-level members as indicated with the angle brackets:
+
+~~~ json
+{
+  "ietf-yang-instance-data:instance-data-set": {
+    "name": "<module-name>@<module-revision>.sid",
+    "description":  ["<description>"],
+    "content-schema": {
+      "module": "ietf-sid-file@2021-11-16"
+    },
+    "content-data": "<replace by SID file contents>"
+  }
+}
+~~~
+
+(Note that the content-data are not actually embedded as a string, but
+as the JSON value of the SID file.  )
```

Comment:
So, it's actually the JSON dict/map, { "foo" : "bar"}, not "{ \"foo\":\"bar\" }", right?
Why not put that into the example then?

Reply:
It seems like "content-data" should be an object, not a string.
This seems like a significant change to the document.

     "content-data": {
          <replace by SID file contents>
      }

Reply:
Yeah, I tried to make the code block a valid piece of JSON and may have obscured what is going on more than what I could compensate with that paragraph. Will update.